example/android/app/build.gradle

```diff
@@ -76,7 +76,9 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    entryFile: "index.js"
+    entryFile: "example/index.js",
+    // TODO: remove once @esemesek's patch is merged
```

Comment:
Maybe more details on identifying the patch, in case you're not the one to fix this.

Reply:
I copied this directly from the slider example, and I'll be honest I don't fully understand why it's necessary

Reply:
turns out it wasn't necessary